### ISSUE TITLE
When keyboard focus moves to the "Select Subscription" combo box, JAWS/NVDA announces the name of the "select subscription" list and the currently focused list item: A11y_VS Code_Appling filter on subscription_Screen reader

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/vscode#245222]
**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**
"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

## GitHub Tags
#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #A11ySev2; #Visual Studio Code Client; #BM-VisualStudioCodeClient-Win32-Jan2024; #WCAG1.3.1; #FTP; #Screen Reader; #AILimited; #NVDA; #JAWS; #Element:Combobox; #STP;

## Environment Details:
Application: VS Code
Visual studio code version: 1.96.3
OS: Windows 11 Enterprise 24H2
Build: 26100.2605
Screen reader
JAWS Version: 2024.2312.53
NVDA Version: 2024.4.1

## Repro Steps

1. Turn on Jaws/NVDA screen reader
2. Launch the VS Code.
3. Install the Azure Machine Learning extension and sign in.
4. Tab until the left toolbar (list of icons). This toolbar has a file icon, search icon, settings, etc.
5. Use the up/down arrow keys to navigate to the "Azure" icon and press enter
6. The Azure panel will now be active next to the toolbar.
7. Tab until the Machine Learning dropdown.
8. Tab till any subscription for example AML V1 Personal subscription.
9. Tab until the "Filter" button and press Enter. It will be a button with a funnel icon.
10. TAB till "Select subscriptions" combo box.
11. TAB till "Select subscriptions" list and press down arrow key to move focus to any list item.
12. Press "Shift + TAB" key to navigate "Select subscriptions" combo box.
13. Observer that JAWS/NVDA announces the name of the "select subscription" list and the currently focused list item.

## Actual Experience
When the keyboard focus moves to the "Select Subscription" combo box after accessing the expanded list with arrow key, JAWS/NVDA announces the name of the "select subscription" list and the currently focused list item.

## Expected Experience
When the keyboard focus moves to the "Select Subscription" combo box, JAWS/NVDA should announce about the "Select Subscription" combo box only. 

## User Impact:
Screen reader user will miss lead as focus lands on "Select Subscription" combo box bug screen reader announcing about the name of the "select subscription" list and the currently focused list item.

## Attachment

NVDA:

![Image](https://github.com/user-attachments/assets/2a479a1d-cd07-462e-ac9d-208941ed7653)

JAWS:

![Image](https://github.com/user-attachments/assets/4e7dcba3-d11b-4d69-86db-f417efcd8bc5)

Code snippet of "select subscription" combo box

![Image](https://github.com/user-attachments/assets/2c6edf2c-c48b-48d1-b65e-fd39cc00f78b)

**Comment by meganrogge** [comment]
I think this is a duplicate of / related to https://github.com/microsoft/vscode/issues/245221

**Comment by TylerLeonhardt** [comment]
This is behaving as expected and we can't change it. Users expect to be able to hit ENTER right as the quick pick opens and so the first item needs to be announced because it is focused by default.

**Comment by TylerLeonhardt** [comment]
@meganrogge it says "Please don't close this bug" how do I close this as "by-design"?

**Comment by kapilvaishna** [comment]
@TylerLeonhardt having list name as "type to narrator down result" creating a different issue https://github.com/microsoft/vscode/issues/245220 issue because when focus move to list we cannot type please check screen shot below and it's misleading the screen reader users. so current name for list should be like "subscription list" for https://github.com/microsoft/vscode/issues/245220 bug.  

Regarding this bug when we navigate to "Select subscriptions" combo box with Shift + TAB" key screen reader should announce the combo box name and role so screen reader user will know that focus is shifted to combo box.

![Image](https://github.com/user-attachments/assets/88cd6f15-ac52-4005-b9d2-c758da0b2af0)
